src/Files.App/Services/Settings/GeneralSettingsService.cs

```diff
@@ -97,6 +97,12 @@ public bool ShowDrivesWidget
 			set => Set(value);
 		}
 
+		public bool ShowNetworkLocationsWidget
+		{
+			get => Get(true);
```

Comment:
What do you think about setting the default on first run? We can enable if the user has existing drives, otherwise, the default can be false.

Reply:
What about displaying an info bar? If the widget is not displayed by default, users may not even know there is it when they have added network locations later.
![image](https://github.com/files-community/Files/assets/66369541/42daa89a-c1df-4551-9b6a-06be48c32033)

Reply:
Adding a button ‘Disable’ might melt then.

Reply:
Can we add "Disable" button on it?

Reply:
> Can we add "Disable" button on it?

Added.

Reply:
> users cannot add network drives and may not even know there is it when they have added network locations later.

There plans to support adding locations from the sidebar.

Reply:
Network shortcuts is not displayed in the sidebar, only displayed in the Network locations widget. Also, network drives is now not displayed in the Drives widget. Should the default still be off?

Reply:
I hear you...

I don't really know the answer in this case, but my guess is that users looking for the widget will be able to find it regardless.

Reply:
FE doesn’t show widget that is empty. So those who don’t have network locations we can hide like sidebar does but currently we don’t have a way to do that so we could leave off by default.
Once we have a way to detect the availability of widget we can show/hide automatically and show by default.

Reply:
Could you share a screenshot of the latest? It looks like network computers are shown in sidebar Drives section in the screenshot in the PR body. Is this correct behavior or is it just out of scope for now?

Reply:
> It looks like network computers are shown in sidebar Drives section in the screenshot in the PR body. Is this correct behavior or is it just out of scope for now?

I think this is not a problem, because it is the same as Explorer.
![image](https://github.com/files-community/Files/assets/66369541/59f1456e-cd79-4da0-bf9c-11aa25a6a62c)